packages/cheerio-crawler/test/xml.test.ts

```diff
@@ -0,0 +1,32 @@
+import type { Server } from 'node:http';
+import type { CheerioCrawlingContext } from '@crawlee/cheerio';
+import { CheerioCrawler } from '@crawlee/cheerio';
+import { runExampleComServer } from '../../../test/shared/_helper';
+
+let serverAddress = 'http://localhost:';
+let port: number;
+let server: Server;
+
+beforeAll(async () => {
+    [server, port] = await runExampleComServer();
+    serverAddress += port;
+});
+
+afterAll(() => {
+    server.close();
+});
+
+describe('CheerioCrawler - XML should be parsed correctly', () => {
+    test('should parse XML', async () => {
+        function handler({ $ }: CheerioCrawlingContext) {
+            const value = $('item').first().find('link').text();
+            expect(value).toBeTruthy();
```

Comment:
assertions from inside request handled will make the request retried 3 times, this does not work as you might think. better to move the assertion outside (just assign the value to some variable defined outside of the scope)